AwesomeCalculator/app/src/main/res/values-uk/strings.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <string name="app_name">Калькулятор</string>
```

Comment:
Is this really translation for UK english? Also be more polite and make as less grammar as possible. Application wording should be neutral.

Reply:
It,s a resourse for Ukraine